sklearn/model_selection/_split.py

```diff
@@ -284,7 +284,16 @@ def __init__(self, n_splits, shuffle, random_state):
 
         self.n_splits = n_splits
         self.shuffle = shuffle
+        # For repr
```

Comment:
What do you mean by this comment? You don't like an explicit random state in the ``__repr__``? Because it's surprising for the user? It would make reproducing easier and not introduce a private variable that is dependent on a global state....

This means that doing ``cv.random_state = 10`` doesn't do anything, though, even though it changes the ``__repr__``.

On the other hand, having a number pop up here if it wasn't specified might be surprising for users that expect this to work like an Estimator object. Which it does not...